modular_newhorizons/master_files/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -0,0 +1,147 @@
+	description = "Adipemcina is a heart medication used for treating high blood pressure, heart failure, and diabetes. Causes vomiting and liver damage when overdosed."
+
+/singleton/reagent/pacifier
+	description = "A mind altering chemical compound capable of suppressing violent tendencies."
+
+/singleton/reagent/rmt
+	description = "A chemical rampantly used by those seeking to remedy the effects of prolonged zero-gravity adaptations."
+
+/* MENTAL */
+
+/singleton/reagent/mental/corophenidate
+	name = "Methylphenidate"
+	description = "Methylphenidate is an ADHD treatment drug that treats basic distractions such as phobias and hallucinations at moderate doses. Withdrawl effects are rare. Side effects are rare, and include hallucinations."
+
+/singleton/reagent/mental/neurostabin
+	name = "Fluvoxamine"
+	description = "Fluvoxamine is safe and effective at treating basic phobias, as well as schizophrenia and muscle weakness at higher doses. Withdrawl effects are rare. Side effects are rare, and include hallucinations."
+
+/singleton/reagent/mental/parvosil
+	name = "Sertraline"
+	description = "Sertraline is cheap, safe, and effective at treating basic phobias, however it does not last as long as other drugs of it's class. Withdrawl effects are uncommon. Side effects are rare."
+
+/singleton/reagent/mental/minaphobin
+	name = "Escitalopram"
+	description = "Escitalopram is expensive, safe and very effective at treating basic phobias as well as advanced phobias like monophobia. A common side effect is drowsiness, and a rare side effect is hallucinations. Withdrawl effects are uncommon."
+
+/singleton/reagent/mental/emoxanyl
+	name = "Duloxetine"
+	description = "Duloxetine is effective at treating basic phobias and concussions. A rare side effect is hallucinations. Withdrawl effects are common."
+
+/singleton/reagent/mental/orastabin
+	name = "Venlafaxine"
+	description = "Venlafaxine is effective at treating basic phobias, monophobia, and stuttering. Side effects are uncommon and include hallucinations. Withdrawl effects are common."
+
+/singleton/reagent/mental/neurapan
+	name = "Risperidone"
+	description = "Risperidone is a potent antipsychotic medication used to treat schizophrenia, stuttering, speech impediment, monophobia, hallucinations, tourettes, and muscle spasms. Side effects are common and include pacifism. Withdrawl symptoms are dangerous and almost always occur."
+
+/singleton/reagent/mental/nerospectan
+	name = "Olanzapine"
+	description = "Olanzapine is a high-strength, expensive antipsychotic medication used to treat schizophrenia, stuttering, speech impediment, monophobia, hallucinations, tourettes, and muscle spasms. Side effects are common and include pacifism. The medication metabolizes quickly, and withdrawl is dangerous."
+
+/singleton/reagent/mental/truthserum
+	description = "This highly illegal, expensive, military strength truth serum is a must have for secret corporate interrogations. Do not ingest."
```

Comment:
"Truth Serum, a refined variant of Paxazide, is an expensive and very unethical psychoactive drug capable of inhibiting defensive measures and reasoning in regards to communication, resulting in those under the effects of the drug to be very open to telling the truth.  This highly illegal, and expensive, military strength truth serum is a must have for secret corporate interrogations, try not to ingest it."